components/storage/blockdevice/COMPONENT_I2CEE/I2CEEBlockDevice.cpp

```diff
@@ -164,3 +196,15 @@ const char *I2CEEBlockDevice::get_type() const
 {
     return "I2CEE";
 }
+
+uint8_t I2CEEBlockDevice::get_paged_device_address(bd_addr_t address)
+{
+    if (!_address_is_eight_bit) {
+        return _i2c_addr;
+    } else {
+        // Use the three least significant bits of the 2nd byte as the page
+        // The page will be bits 2-4 of the user defined addresses.
+        return _i2c_addr | ((address & 0x0700u) >> 7u);
+    }
+}
+
```

Comment:
Think you've gone overboard and now have an excess blank line :)

Reply:
Oops, I didn't know where you wanted the NL.
Where exactly did you want the NL?
Thanks.

Reply:
It's possible I'm being misled by GitHub's view, but I think originally you had an unterminated final `}` line. I think you went to the bottom and pressed "Enter" twice rather than once, so you now have an excess blank line.